schematics/ng-add/setup-project.spec.ts

```diff
@@ -14,15 +11,18 @@ import {createTestApp} from '../utils/testing';
     let runner: SchematicTestRunner;
     let log: string[] = [];
 
-    async function createAppWithOptions(appOptions = {}): Promise<{ tree: UnitTestTree, project: WorkspaceProject }> {
+    async function createAppWithOptions(appOptions = {}):
+        Promise<{ tree: UnitTestTree, project: workspaces.ProjectDefinition }> {
       // 'app' is the default application, so we're not passing '--project' option
       const options: Schema = {project: projectName};
       let tree = await createTestApp(runner, appOptions);
       tree = await runner.runSchematicAsync('ng-add-setup-project', options, tree).toPromise();
-
-      const workspace = getWorkspace(tree);
-      const project = getProject(workspace, projectName);
-      return {tree, project};
+      const workspace = await getWorkspace(tree);
+      const project = workspace.projects.get(projectName);
+      return {tree,
+              // @ts-ignore TODO: types is not compatible because of ngx-build-plus have old dependency on
```

Comment:
Shouldn't you use `@ts-expect-error` here so that it gets found when `ngx-build-plus` does get updated?